### ISSUE TITLE
Strange crash (RuntimeException) 

### ISSUE THREAD

**Issue by LordKered** [wuseal/JsonToKotlinClass#357]
AS: 4.2.1 
Plugin: 3.7.0

_Occurred crash:_
```
{
  "page": {
    "href": "a"
  },
  "_page": {
    "href": "a"
  }
}
```
_Success:_
```
{
  "page": {
    "href": "a"
  },
  "_page": {
    "href": "b"
  }
}
```

_Success:_
```
{
  "page": {
    "href": "a"
  },
  "_page1": {
    "href": "a"
  }
}
```

**Comment by wuseal** [comment]
@LordKered Confirmed, Thanks for your feedback, Will fix it in 3.7.1

**Comment by wuseal** [comment]
Fixed in 3.7.1
